### PR TITLE
Release highlights 0.9

### PR DESCRIPTION
**Pull request by pebrc**
Add a more high level summary of important changes.

### CODE REVIEW

File: docs/release-notes/highlights-0.9.0.asciidoc [hunk 1]
```diff
@@ -0,0 +1,33 @@
+[[release-highlights-0.9.0]]
+== 0.9.0 Release Highlights
+
+[float]
+=== Kubernetes Distribution support
```
Comment (anyasabo) [inline docs/release-notes/highlights-0.9.0.asciidoc]:
```suggestion
=== Kubernetes Distribution Support
```

consistency change

File: docs/release-notes/highlights-0.9.0.asciidoc [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+In addition to Google Kubernetes Engine (GKE) and vanilla Kubernetes ECK now also works on
```
Comment (anyasabo) [inline docs/release-notes/highlights-0.9.0.asciidoc]:
```suggestion
In addition to Google Kubernetes Engine (GKE) and vanilla Kubernetes, ECK now also works on
```

File: docs/release-notes/highlights-0.9.0.asciidoc [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+Azure Kubernetes Service (AKS), Amazon Container Service for Kubernetes (EKS) and with the some restrictions for APM server
```
Comment (anyasabo) [inline docs/release-notes/highlights-0.9.0.asciidoc]:
```suggestion
Azure Kubernetes Service (AKS), Amazon Elastic Kubernetes Service (EKS) and
```

File: docs/release-notes/highlights-0.9.0.asciidoc [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+on Red Hat OpenShift 3.11+
```
Comment (anyasabo) [inline docs/release-notes/highlights-0.9.0.asciidoc]:
```suggestion
Red Hat OpenShift 3.11+ (with some restrictions on APM server functionality)
```

File: docs/release-notes/highlights-0.9.0.asciidoc [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+In addition to self-signed certificates which ECK uses as a default, we now support user provided certificates for
+HTTP connections to Elasticsearch, Kibana or APM. Configure them via a regular k8s TLS secrets and
+reference them in the Elastic resource of choice in the new TLS options attribute.
+
+[float]
+=== User Provided Pod Templates
+
+This release introduces support for user provided pod templates in Elasticsearch, Kibana and APM server. This allows you configure, for example,
+your own init containers to run code before Elasticsearch, Kibana or APM start up. Or you can use the pod template to mount
+additional volumes with configuration files e.g. to configure SAML , synonym files or specifiy `imagePullSecrets` for your private Docker registry.
```
Comment (anyasabo) [inline docs/release-notes/highlights-0.9.0.asciidoc]:
```suggestion
additional volumes with configuration files e.g. to configure SAML or synonym files, or to specify `imagePullSecrets` for your private Docker registry.
```

File: docs/release-notes/highlights-0.9.0.asciidoc [hunk 6]
```diff
@@ -0,0 +1,33 @@
+[[release-highlights-0.9.0]]
+== 0.9.0 Release Highlights
+
+[float]
+=== Kubernetes Distribution Support
+
+In addition to Google Kubernetes Engine (GKE) and vanilla Kubernetes, ECK now also works on
+Azure Kubernetes Service (AKS), Amazon Elastic Kubernetes Service (EKS) and
+Red Hat OpenShift 3.11+ (with some restrictions on APM server functionality)
```
Comment (thbkrkr) [inline docs/release-notes/highlights-0.9.0.asciidoc]: Missing final dot.

Comment (thbkrkr) [inline docs/release-notes/highlights-0.9.0.asciidoc]: I'm wondering if it's important to write APM server with a **s** in uppercase (applicable below).

File: docs/release-notes/highlights-0.9.0.asciidoc [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+[float]
+=== APM Server
+
+ECK 0.9.0 brings support for a new custom resource that allows you to deploy APM server on Kubernetes.
+The ECK operator takes care of linking it up to an Elasticsearch output, if desired, as well.
+
+[float]
+=== User Provided HTTP Certificates
+
+In addition to self-signed certificates which ECK uses as a default, we now support user provided certificates for
```
Comment (alaudazzi) [inline docs/release-notes/highlights-0.9.0.asciidoc]: user-provided certificates

File: docs/release-notes/highlights-0.9.0.asciidoc [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+HTTP connections to Elasticsearch, Kibana or APM. Configure them via a regular k8s TLS secrets and
```
Comment (alaudazzi) [inline docs/release-notes/highlights-0.9.0.asciidoc]: Configure them with regular k8s TLS secrets

File: docs/release-notes/highlights-0.9.0.asciidoc [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+reference them in the Elastic resource of choice in the new TLS options attribute.
+
+[float]
+=== User Provided Pod Templates
+
+This release introduces support for user provided pod templates in Elasticsearch, Kibana and APM server. This allows you to configure, for example,
+your own init containers to run code before Elasticsearch, Kibana or APM start up. Or you can use the pod template to mount
+additional volumes with configuration files e.g. to configure SAML or synonym files, or to specify `imagePullSecrets` for your private Docker registry.
+
```
Comment (alaudazzi) [inline docs/release-notes/highlights-0.9.0.asciidoc]: > This release introduces support for user-provided Pod templates in Elasticsearch, Kibana and APM server. For example, this allows you to configure your own init containers to run code before Elasticsearch, Kibana or APM start up. Or you can use the Pod template to mount additional volumes with configuration files, for example to configure SAML or synonym files, or to specify `imagePullSecrets` for your private Docker registry.

File: docs/release-notes/highlights-0.9.0.asciidoc (hunk 8 above)
Comment (pebrc) [reply docs/release-notes/highlights-0.9.0.asciidoc]: just one, removing the 's'

**Pull request merged**